docs/visualization/basic_api_concept.md

```diff
@@ -0,0 +1,41 @@
+# Basic APIs Concept
+
+## The Visualization Policy of CARET
```

Comment:
It might better to remove `The Visualization Policy of CARET` because figure 1 is not prepared yet.
It looks prefered to add `visualization policy` after merging current documentation.